test/test_jit.py

```diff
@@ -13440,6 +13440,16 @@ def call(self):
         self.assertTrue(isinstance(f, torch.jit.ScriptModule))
         self.assertTrue(isinstance(f.call(), property))
 
+    def test_monkeytype(self):
```

Comment:
Why isn't this test case in same file as those functions being tested?

Reply:
Now moved.